src/inference/gibbs.jl

```diff
@@ -36,25 +36,38 @@ end
 alg = Gibbs(HMC(0.2, 3, :v1), PG(20, :v2))
 ```
 
+One can also pass the number of iterations for each Gibbs component using the following syntax:
+- `alg = Gibbs((HMC(0.2, 3, :v1), n_hmc), (PG(20, :v2), n_pg))`
+- `alg = Gibbs(HMC(0.2, 3, :v1) => n_hmc, PG(20, :v2) => n_pg)`
```

Comment:
Could we support only one syntax? Maybe only pairs as it more clearly illustrates the key-value-like relationship (but I'm fine with tuples as well)? I don't think there's a big advantage in supporting both syntaxes, and only supporting one would keep the API simpler and it would be clear what's the recommended/official way.

Reply:
I'm fine with keeping only the `Pair` version but Hong prefers the `Tuple` version.

Reply:
Then let's only use `Tuple`?